tasks/update-code/copy.yml

```diff
@@ -1,3 +1,6 @@
 ---
+- name: ANSISTRANO | SCP | Create release folder
```

Comment:
Is this needed for copy? I am pretty sure copy will create the folder if it does not exist. It does not hurt to have it, though